### PR TITLE
Update textacy version number

### PR DESCRIPTION
**Pull request by gryBox**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -40,6 +40,7 @@ requirements:
     - scikit-learn >=0.17
     - spacy >=1.7.0,<2.0.0
     - unidecode >=0.04.19
+    - mwparserfromhell>=0.4.4
```
Comment (rmax) [inline recipe/meta.yaml:43]: You can remove this package as it's neither listed in textacy's setup or available in conda.

And you can add `tqdm>=4.11.1`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {% set name = "textacy" %}
-{% set version = "0.3.4" %}
+{% set version = "0.4.1" %}
 {% set sha256sum = "56117d2e2549d33eafc91735e20c7e9e1a955cbe271788d50610adb2b860e46a" %}
```
Comment (rmax) [inline recipe/meta.yaml:3]: You need to update the sha256.

Comment (gryBox) [reply recipe/meta.yaml:3]: where do you get this?

Comment (rmax) [reply recipe/meta.yaml:3]: You can find it in the pypi.io page: https://pypi.org/project/textacy/#files

**Comment by rmax** [comment]
Deprecated by #6

**Pull request closed without merging**